platform/security/certificate/src/main/java/org/codice/ddf/certificate/JKSManager.java

```diff
@@ -0,0 +1,207 @@
+     * @param keyStorePassword password for the keystore
+     * @param alias            alias of the certificate to delete
+     *                         -------------------------------------------------------------------
+     */
+
+    public static void removeCertFromKeyStore(String keyStorePath, String keyStorePassword,
+            String alias) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
+            jks.load(new FileInputStream(keyStorePath), keyStorePassword.toCharArray());
+            jks.deleteEntry(alias);
+            jks.store(new FileOutputStream(keyStorePath), keyStorePassword.toCharArray());
+        } catch (CertificateException | NoSuchAlgorithmException | KeyStoreException | IOException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * ---------------------------------------------------------------------
+     * Method: saveX509toNewKeystore
+     * <p/>
+     * Purpose:  Creates a new keystore file and saves an X509 certificate to it.
+     * <p/>
+     * Pre-condition:  None
+     * <p/>
+     * Post-condition: The keystore file that is output will contain the certificate passed in as a parameter
+     *
+     * @param x509       certificate to be saved to the new keystore
+     * @param privateKey the key to be associated with the alias
+     * @param fileName   the name of the file to save
+     * @param alias      alias name
+     * @param password   the password to protect the key
+     *                   -------------------------------------------------------------------
+     */
+
+    public static void saveX509toNewKeystore(X509Certificate x509, PrivateKey privateKey,
+            String fileName, String alias, String password) {
+        try {
+            privateKey.getEncoded();
+            KeyStore keyStore = KeyStore.getInstance("jks");
+            keyStore.load(null, null);
+            keyStore.setKeyEntry(alias, privateKey, password.toCharArray(),
+                    new java.security.cert.Certificate[] {x509});
+            keyStore.store(new FileOutputStream(new File(fileName)), password.toCharArray());
+        } catch (KeyStoreException | CertificateException | NoSuchAlgorithmException | IOException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * ---------------------------------------------------------------------
+     * Method: createNewKeyStore
+     * <p/>
+     * Purpose:  Creates a new keystore file with the given name
+     * <p/>
+     * Pre-condition:  None
+     * <p/>
+     * Post-condition: The keystore file is created
+     *
+     * @param keyStoreName the name of the keystore file that will be output
+     * @param password     the password for the new keystore
+     *                     -------------------------------------------------------------------
+     */
+
+    public static void createNewKeyStore(String keyStoreName, String password) {
```

Comment:
* Rename `keyStoreName` to `keyStorePath` to be consistent.
* Should we create a new instance of this class (JksKeyStore) and return it? This would help with other usage scenarios (see `saveX509toNewKeystore()` comment).